### PR TITLE
LPS-104697 Implement tests

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/odata/entity/v1_0/ContentPageEntityModel.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+				locale -> Field.CREATE_DATE),
+			new DateTimeEntityField(
+				"dateModified",
+				locale -> Field.getSortableFieldName(Field.MODIFIED_DATE),
+				locale -> Field.MODIFIED_DATE),
+			new StringEntityField(
+				"title",
+				locale -> LocalizationUtil.getLocalizedName(
+					Field.TITLE, LocaleUtil.toLanguageId(locale))));
+	}
```
Comment (javierdearcos) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/odata/entity/v1_0/ContentPageEntityModel.java:47]: Did we check if there are any more fields that make sense to add to the entity model?

**Review by javierdearcos — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**